static/app/components/sentryAppComponentIcon.tsx

```diff
@@ -31,4 +31,5 @@ const SentryAppAvatarWrapper = styled('span')<{isDark: boolean; isDefault: boole
   color: ${({isDark}) => (isDark ? 'white' : 'black')};
   filter: ${p => (p.isDark && !p.isDefault ? 'invert(1)' : 'invert(0)')};
   line-height: 0;
+  flex-shrink: 0;
```

Comment:
Sadly we have to do this in two places because there's two components that look the same: one for regular integrations and the other for custom integrations